.github/workflows/test.yml

```diff
@@ -28,7 +28,9 @@ jobs:
         shell: pwsh
         run: Set-DisplayResolution -Width 1920 -Height 1080 -Force
       - name: Build with Gradle
-        run: ./gradlew ${{ matrix.gradle-task }} --no-parallel --no-daemon --console=plain -i
+        uses: gradle/gradle-build-action@6095a76664413da4c8c134ee32e8a8ae900f0f1f
```

Comment:
Why are you using a specific commit instead of a specific version?

Reply:
@BorisOsipov I don't know, this code is automatically generated by Github actions wizard.
If you create a new project, go to "Actions" -> "Java with Gradle", it generates such a script by default:

```yml
# This workflow uses actions that are not certified by GitHub.
# They are provided by a third-party and are governed by
# separate terms of service, privacy policy, and support
# documentation.
# This workflow will build a Java project with Gradle and cache/restore any dependencies to improve the workflow execution time
# For more information see: https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-java-with-gradle

name: Java CI with Gradle

on:
  push:
    branches: [ "master" ]
  pull_request:
    branches: [ "master" ]

permissions:
  contents: read

jobs:
  build:

    runs-on: ubuntu-latest

    steps:
    - uses: actions/checkout@v3
    - name: Set up JDK 11
      uses: actions/setup-java@v3
      with:
        java-version: '11'
        distribution: 'temurin'
    - name: Build with Gradle
      uses: gradle/gradle-build-action@67421db6bd0bf253fb4bd25b31ebb98943c375e1
      with:
        arguments: build
```

Reply:
@BorisOsipov I replace it with `gradle/gradle-build-action@v2`. I hope it's effectively the same thing.